Install and configure RuboCop for the project.

archanaserver opened the pull request:
Added and configured RuboCop for the project. Modify configuration to exclude patterns for files and cops that we don't want RuboCop to treat as problems.

Resolves #5 Set up RuboCop.

Ron-Lavi commented on Gemfile.lock:
```diff
@@ -68,6 +68,7 @@ GEM
       tzinfo (~> 2.0)
     addressable (2.8.1)
       public_suffix (>= 2.0.2, < 6.0)
+    ast (2.4.2)
```
Can you add this file to .gitignore?

Ron-Lavi reviewed (commented):
Can you do a rebase after @girijaasoni PR got in?

Can you also add a CI for rubocop, e.g.: https://github.com/LiorKGOW/My-Guitar-Store/blob/main/.github/workflows/rubocop.yml

archanaserver committed:
Set up Rubocop

- Add and configure RuboCop for the project. Modify configuration to
exclude patterns for files and cops that we don't want RuboCop
to treat as problems.

- Also added RuboCop CI workflow.

Resolves #5 Installed and configured RuboCop for the project.

Ron-Lavi reviewed (approved):
Thanks @archanaserver!
